eras/alonzo/impl/src/Cardano/Ledger/Alonzo.hs

```diff
@@ -89,7 +89,7 @@ instance (Crypto c, DSignable c (Hash c EraIndependentTxBody)) => API.ApplyBlock
 instance Crypto c => API.CanStartFromGenesis (AlonzoEra c) where
   type AdditionalGenesisConfig (AlonzoEra c) = AlonzoGenesis
 
-  initialState = API.initialStateFromGenesis extendPPWithGenesis
+  fromShelleyPParams ag = translatePParams ag . translateEra' () . translateEra' ()
```

Comment:
I would not opposite getting rid of another tick. I am not sure about the `V` suffix, since `Void` is gone by that point. Maybe `translateEraTotal`, or something else that is descriptive. Let's do it in a separate PR though. Wanna create a short ticket about it?